Middlewares/Handlers/ErrorHandler.js

```diff
@@ -21,6 +21,10 @@ class ErrorHandler {
 			statusCode = 500;
 			message = 'Internal Server Error';
 		}
+		else if(err.name === 'AlreadyExistError') {
+			statusCode = 404,
```

Comment:
```suggestion
			statusCode = 409,
```

404 is for not found